contracts/tokenomics/staking/README.md

```diff
@@ -17,40 +17,65 @@ README has updated with new messages (Astroport v1 messages follow).
 
 ## ExecuteMsg
 
-### `enter`
+### `receive`
 
-Deposits token to get share token amount.
+CW20 receive msg.
 
 ```json
 {
-  "enter": {
-    "amount": "123"
+  "receive": {
```

Comment:
Documentation is not clear, please use following example, so users know how to use it:
https://docs.terraswap.io/docs/howto/freq_used_msg/#withdraw